style.css

```diff
@@ -85,20 +80,3 @@ form{
     background-color: rgb(127, 177, 225);
 }
 
-@media only screen and (max-width: 480px) {
-    nav{
-        display: flex;
-        flex-direction: column;
-        min-width: 100%;
-    }
-    .img-div{
-        width: 100%;
-    }
-    header{
-        display: flex;
-        flex-direction: column;
-    }
-    #hero {
-      top: 200px;
-    }
-  }
```

Comment:
Inline media need some correction I think.

Reply:
- [ ] Good work using inline comments to provide feedback 👏🏾 . There are a few more things we need to do to add value to our review.
 - You should always phrase comments politely, as suggestions instead of orders. Words like `Please`, `Kindly`, and` I suggest` help a lot in making your comments sound polite. 🤔 
 
 - Please, always include an explanation, a "why", for each requested change. 😄 
 
 - Your comment should also be clear and easy to understand.  You should state what correction you think should be done. 🤝

Reply:
- [ ] Great job 👏 making your comments sound more friendly and polite but once again as requested by the previous reviewer please make sure to add an explanation as to "why" you requested a certain change. This will help the student grow professionally and know why certain things should be done a certain way.
-  [ ] [OPTIONAL] You can also consider adding an emoji to your comments this makes it seem more playful & less serious.